tmail-backend/plugins/mail-reporting/src/main/java/com/linagora/tmail/cassandra/CassandraMailReportGenerator.java

```diff
@@ -93,15 +86,13 @@ private PreparedStatement prepareInsert() {
             .value(RECIPIENT, bindMarker(RECIPIENT))
             .value(SUBJECT, bindMarker(SUBJECT))
             .value(DATE, bindMarker(DATE))
-            .value(DATE_YEAR_MONTH, bindMarker(DATE_YEAR_MONTH))
             .usingTtl(TTL)
             .build());
     }
 
     private PreparedStatement prepareSelectByDateBetween() {
         return session.prepare(selectFrom(TABLE_NAME)
-            .all()
-            .whereColumn(DATE_YEAR_MONTH).isEqualTo(bindMarker(DATE_YEAR_MONTH))
+            .all().allowFiltering()
```

Comment:
We do not need allowFiltering without where clause

Reply:
Do you mean that we will filter by java code after Cassandra fetches all, 
or do we not need to query by start and end date? (remove query by date, and remove "duration" in webadmin route)

Reply:
We need allowFiltering for
```
.whereColumn(DATE).isGreaterThanOrEqualTo(bindMarker(DATE_START))
            .whereColumn(DATE).isLessThanOrEqualTo(bindMarker(DATE_END))
```

Reply:
> Do you mean that we will filter by java code after Cassandra fetches all

Yes.

Easier: it avoids timeouts